5_data_munge/src/qaqc_daily_data.R

```diff
@@ -0,0 +1,62 @@
+############ Function to read the daily temperature and site ind data #######
+############  Create longitude, latitude, doy of the year bins ################
+qaqc_daily_temp_site_data <- function(temp_in_ind, site_in_ind, out_ind) {
+
+  # reading the daily temperature data-in, removing NA dates, 
+  # and creating day of the year column.
+  #creating doy bins with 6 days intervals.
+  doy_lim <- 6
+  daily_dat_mod <- readRDS(sc_retrieve(temp_in_ind), remake_file = '5_data_munge.yml') %>%
```

Comment:
See the pattern of scipiper usage in other scripts. The remake_file here should be `getters.yml`, and you'll need to add the appropriate targets to `getters.yml`.